pkg/api/controller.go

```diff
@@ -117,8 +117,9 @@ func (c *Controller) Run() error {
 
 	engine.Use(
 		SessionLogger(c),
-		handlers.RecoveryHandler(handlers.RecoveryLogger(c.Log),
```

Comment:
Why remove the handlers.RecoveryLogger(c.Log)?

Reply:
Keeping that causes panic backtrace to not show.

Reply:
So where should it be shown, in stdout or in the logfile?
I assume in whatever place is configured via zot config, so where is that reference to the logger in this specific case?

Reply:
Shouldn't changing true to false be enough to solve the issue?

Reply:
Doesn't appear to be enough.